File: 217-contains-duplicate/contains-duplicate.cs
public class Solution {
    public bool ContainsDuplicate(int[] nums) {
        HashSet<int> seen = new HashSet<int>();
        foreach (int num in nums)
        {
            if (seen.Contains(num))
                return true;
            seen.Add(num);
        }

        return false;
    }
}

File: 217-contains-duplicate/contains-duplicate.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        HashSet hs = new HashSet();

        for (int i = 0; i < nums.length; i++)
        {
            int num = nums[i];
            if (hs.contains(num))
            {
                return true;
            }

            hs.add(num);
        }

        return false;
    }
}